generator/php/resources/templates/README.mustache

```diff
@@ -8,10 +8,7 @@ More information: [https://developers.criteo.com/marketing-solutions/docs/php-ap
 
 This PHP package is automatically generated by the [OpenAPI Generator](https://openapi-generator.tech) project:
 
-- API version: {{apiVersion}}
-{{#apiVersion}}
-- Package version: {{apiVersion}}
-{{/apiVersion}}
+- API version: {{artifactVersion}}
```

Comment:
the artifactVerison corresponds more to the version of the package right?

Reply:
"-Package version : {{artifactVersion}}"